standalone/install.ts

```diff
@@ -1,20 +1,28 @@
 import * as http from "http";
 import * as request from "request";
-import {getPlatformFolderName, PACT_STANDALONE_VERSION} from "../src/pact-standalone";
 
 const path = require("path");
 const fs = require("fs");
+const urljoin = require("url-join");
 const decompress = require("decompress");
 const tar = require("tar");
 const chalk = require("chalk");
+const sumchecker = require("sumchecker");
+
+export const PACT_STANDALONE_VERSION = "1.44.0";
+export const PACT_BINARY_LOCATION = process.env.PACT_BINARY_LOCATION ||
```

Comment:
I thought we were going with the entry in `package.json` as per #88?

Reply:
Yeah, still need to figure that out as I'm not sure how to get the package.json of the containing project...

Reply:
Not sure this is possible.  Seems that it only works if the thing runs through NPM itself, like (`npm run whatever`).  I might just stick to environment variables cause I think it just might be simpler for now.  We can always adapt it later.